index.js

```diff
@@ -57,6 +57,8 @@ module.exports = {
       }
     }
 
+    app.import('vendor/ember-prism.js');
```

Comment:
What is this for?

Reply:
We need to make prism.js go into "manual" mode, i.e. not apply highlighting to any code elements by default on `DOMContentLoaded`, see https://github.com/PrismJS/prism/blob/6068bf18e53222a1ee712b073a9864413fefeef3/prism.js#L531-L541. Mangling with the DOM on code blocks rendered with Glimmer will cause exceptions when later Glimmer tries to update "its" DOM. Which is the case for the dynamic updates enabled here. See also the comment in the imported file.

As we let Glimmer render the *already highlighted* code (here: https://github.com/shipshapecode/ember-prism/pull/33/files#diff-28fe0c00eea01c10c0b6b9d67c35f741R31), there is no need for this highlight by default behaviour!